app/tests/fixtures/events.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture
+def generic_event():
```

Comment:
[I don't always use fixtures, but when I do](https://knowyourmeme.com/memes/the-most-interesting-man-in-the-world), I like to name them according to the following template: "def a_<fixture>():".

Here, I would call this "def an_event()", and when we will need more fixtures, we can add other ones like "def an_event_with_this_thing_in_particular():"

What do you think ?

Reply:
And there is some type hinting missing :-)